### DISCUSSION TITLE
New boards

### DISCUSSION THREAD

**Discussion by LaZsolt** [dbuezas/lgt8fx#187] [General]
I found two new MiniEVB boards on Aliexpress
| LQFP32 MCU with USB-C and 16 MHz crystal | LQFP48 MCU with USB-C |
| - | - |
| ![LGT8F328P-MiniEVB-TYPE-C-with-crystal-top-1024](https://user-images.githubusercontent.com/60486174/144754730-bf021631-37b6-4bc7-93d0-518fc2291919.jpg) | ![LQFP48-MiniEVB-top-1024](https://user-images.githubusercontent.com/60486174/144754717-c19bdfd9-679c-4afb-b3e5-c8e00ada76f0.jpg) |

**Comment by ithinkido** [comment]
I have one of the purple ones. I managed to flash a blink sketch on it , Worked great. Sadly I tried the EEPROM write example on in and it looks like that corrupted the boot loader and the board is not reachable. Waiting on delivery of a second board to use as ISP to flash new boot loader.

**Reply by FishmannJ** [reply]
Stalo se mi to samé. Je nějaká rada? Chci desku použít k řízení laserového gravírování. Jak na flash gbrl?

**Comment by trimarco232** [comment]
Hi,
now at aliexpress, a 48 pins, purple board with crystal
https://fr.aliexpress.com/item/1005004317038120.html?spm=a2g0o.productlist.0.0.6fdd7840UMCqiH&pdp_ext_f=%7B%22sku_id%22%3A%2212000028730955312%22%7D&pdp_npi=2%40dis%21EUR%213.02%213.02%21%21%213.76%21%21%402100bdec16604000419162914e3d46%2112000028730955312%21sea&curPageLogUid=xL1Z9hERc549
pin# 15 of the 48 pin devices is called AC0P / OC3A it has no PORT
if you want to use it as a simple io, you must  : 
- as an input, go through the analog comparator 0
- as an output  : 
if you use timer3 for pwm :  use a duty = 0 for LOW  , or a duty = the period for High
take care that in the case of duty = 0 (OC3A=0) one won't obtain a real LOW state, but a short glitch at each period (not visible if driving a LED)
if you don't use timer3 for pwm, use the following code : 
in setup : 
`PMX0 |= _BV(WCE) ;  // jumps the lock`
`PMX1 |= _BV(C3AC) ;  // put the pin on OC3A`
`TCCR3A = _BV(COM3A1) ; // COM1A1 = 1 : Set OC1A/OC1B on compare match`
then for HIGH : 
`TCCR3A |= _BV(COM3A0) ; // COM1A0 = 1 : Set OC1A/OC1B on compare match`
`TCCR3C = _BV(FOC3A) ;  // Force Output Compare for Channel A `
and for LOW : 
`TCCR3A &= ~_BV(COM3A0) ; // COM1A0 = 0 : Clear OC1A/OC1B on compare match`
`TCCR3C = _BV(FOC3A) ;  // Force Output Compare for Channel A`
